scripts/convert-docs.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { writeFile } = require('fs/promises');
+const apiDocs = require('../src/api-docs');
+
+(async () => {
+  // Remove redundant `default` property
+  const docsObj = apiDocs.default || apiDocs;
+
+  // Remove obsolete properties
+  delete docsObj.components.securitySchemes;
```

Comment:
We use this to indicate that it may use an apikey, but if it is obsolete, should we use the way to indicate this instead of dynamically fixing the api-docs before linting?

Reply:
Our current workflow involves removing this key manually, that's why it is marked as obsolete.
An option would be to leave it there for the developer to validate, but only remove it when executing the production workflow to keep our current process.

@pedroferreira1, do you have any comment about this?

Reply:
I think that if we are excluding the apikey from the security schemes when "building" the api-docs, we should not have the apikey on the docs in the first place.
This would make this script essentially unneeded, so we can just serve the api-docs itself.

Reply:
@r4mmer yes, we have this to indicate we support api key. However, back when I created the first docs page, I didn't find a nice way to make it clear that the api key was optional. Since this is a feature that most users don't use (and maybe don't even know that exists), I decided to remove it from the docs page because it was confusing having it in every API.

We can discuss the best approach here but that the way I've been doing to prevent a misunderstanding.

Reply:
On [9f2ac73](https://github.com/HathorNetwork/hathor-wallet-headless/pull/327/commits/9f2ac7363b8f9c730d99feeac429be93197554c7) a check was implemented to fetch the API Key configuration directly from the config file at runtime.

No arbitrary removal is necessary anymore.